src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -90,6 +90,8 @@ static Microsoft::Console::TSF::Handle& GetTSFHandle()
 
 namespace winrt::Microsoft::Terminal::Control::implementation
 {
+    Windows::UI::ViewManagement::AccessibilitySettings TermControl::_accessibilitySettings{};

```

Comment:
ah, I just realized. We can't do this. We cannot have static globals that _instantiate WinRT classes_ during DLL load time.

This will break the safety of DLL initialization, _and_ it will break the initialization of TermControl.dll on earlier versions of Windows (like the one VS uses) even if they aren't actually using TermControl itself.